build-recipe-docker

```diff
@@ -56,9 +56,12 @@ recipe_build_docker() {
     touch $BUILD_ROOT/etc/resolv.conf
 
     base_image_path=
+    base_image_tag=$(grep "^\s*FROM" "$RECIPEFILE" | head -n 1 | cut -d" " -f2)
```

Comment:
you're checking only the first "FROM" statement, but my understanding is that the first image needs to be non-scratch, only further on layers (multistages) need to be scratch?

Reply:
ah, you're having a signle stage with the first stage going from scratch, okay so thats a special case. :+1: